app/src/components/fields/ReadMoreField.tsx

```diff
@@ -28,7 +28,7 @@ export const ReadMoreField: React.FC<IReadMoreFieldProps> = (props) => {
   const willTruncateText = sanitizedText.length > maxCharLength;
 
   if (!willTruncateText) {
-    return <span>{sanitizedText}</span>;
+    return <Typography color="textSecondary">{sanitizedText}</Typography>;
```

Comment:
This `<span>` is already renderted inside a `<Typography color="textSecondary">` (see below).

Reply:
I don't think this span was wrapped in typography tags before. This return here, as the code is setup currently, skips everything below entirely, so it just ends up returning a <span> with no styling.